source/component/PasDoc_Items.pas

```diff
@@ -754,7 +754,8 @@   TPasProperty = class(TPasItem)
   TCIOType = (CIO_CLASS, CIO_PACKEDCLASS,
     CIO_DISPINTERFACE, CIO_INTERFACE,
     CIO_OBJECT, CIO_PACKEDOBJECT,
-    CIO_RECORD, CIO_PACKEDRECORD );
+    CIO_RECORD, CIO_PACKEDRECORD,
+    CIO_TYPE {helper only});
```

Comment:
Let's call it `CIO_TYPEHELPER` (instead of just `CIO_TYPE`), to be more obvious. 

Then the comment `{helper only}` will also not be necessary -- the `CIO_TYPEHELPER` name already communicates everything :)

Reply:
Hi Michalis,

It's been some time. Sorry for the delay.

So, I don't think it would make sense to change it to CIO_TYPEHELPER because for the record helper, CIO_RECORD is used. Here it is the type that is targeted by the helper, not the helper itself.

Regards